flink-connectors/flink-connector-kinesis/pom.xml

```diff
@@ -56,6 +58,18 @@ under the License.
 		</dependency>
 
 		<dependency>
+		    <groupId>org.apache.httpcomponents</groupId>
+		    <artifactId>httpclient</artifactId>
+		    <version>${apache.httpclient.version}</version>
+		</dependency>
+
+		<dependency>
+		    <groupId>org.apache.httpcomponents</groupId>
+		    <artifactId>httpcore</artifactId>
+		    <version>${apache.httpcore.version}</version>
+		</dependency>
```

Comment:
To clarify a bit more:

- hadoop-aws 2.7 depends on http-components 4.2.x
- aws-java-sdk-kinesis 1.10.71, a core dependency of flink-kinesis-connector, depends on http-components 4.3.x.

Thus, we should include http-components 4.3 into flink-kinesis-connector so that users won't worry about deciding http-components version if they use hadoop-aws and flink-kinesis-connector at the same time.